README.md

```diff
@@ -165,10 +168,10 @@ Hi, I'm Wingy. I made this app. My website is [samwing.dev](https://samwing.dev)
 <table>
 <tr>
     <td align="center">
-        <a href="https://github.com/Wingysam">
```

Comment:
Apologies. First time submitting a comment on a PR on mobile. These are the lines specifically to revert before merging. Thanks.

Reply:
It’s OK, I was worried about this previously but it resolves itself when @Wingysam merges.